src/Uno.UI/UI/Xaml/Controls/MenuFlyout/MenuFlyoutPresenter.cs

```diff
@@ -1,13 +1,958 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Runtime.InteropServices.WindowsRuntime;
+using Windows.Devices.Input;
+using Windows.Foundation;
+using Windows.Foundation.Collections;
+using Windows.System;
+using Windows.UI.Input;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Automation;
+using Windows.UI.Xaml.Automation.Peers;
+using Windows.UI.Xaml.Controls;
 using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Media.Animation;
 
 namespace Windows.UI.Xaml.Controls
 {
-	public  partial class MenuFlyoutPresenter : global::Windows.UI.Xaml.Controls.ItemsControl
+	public partial class MenuFlyoutPresenter : ItemsControl, IMenuPresenter
 	{
-		public global::Windows.UI.Xaml.Controls.Primitives.MenuFlyoutPresenterTemplateSettings TemplateSettings { get; } = new MenuFlyoutPresenterTemplateSettings();
+		// Can be negative. (-1) means nothing focused.
+		internal int m_iFocusedIndex;
 
-		public MenuFlyoutPresenter() : base()
+		// Weak reference to the menu that ultimately owns this MenuFlyoutPresenter.
+		private WeakReference m_wrOwningMenu;
+
+		// Weak reference to the parent MenuFlyout.
+		private WeakReference m_wrParentMenuFlyout;
+
+		// Weak reference to the owner of this menu.
+		// Only populated if this is the presenter for an ISubMenuOwner.
+		private WeakReference m_wrOwner;
+
+		// Weak reference to the sub-presenter that was created by a child menu owner.
+		private WeakReference m_wrSubPresenter;
+
+		// Whether ItemsSource contains at least one ToggleMenuFlyoutItem.
+		private bool m_containsToggleItems;
+
+		// Whether ItemsSource contains at least one MenuFlyoutItem with an Icon.
+		private bool m_containsIconItems;
+
+		// Whether ItemsSource contains at least one MenuFlyoutItem or ToggleMenuFlyoutItem with KeyboardAcceleratorText.
+		private bool m_containsItemsWithKeyboardAcceleratorText;
+
+		// UNO TODO
+		// private bool m_animationInProgress;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5205492)